controllers/user_controller.go

```diff
@@ -0,0 +1,60 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+)
+
+type UserController interface {
+	Info(context *gin.Context)
+}
+
+type userController struct {
+}
+
+func NewUserController() UserController {
+	return &userController{}
+}
+
+func (ctrl *userController) Info(context *gin.Context) {
+	// Id refers to MongoDB user object ID. Specificty refers to the verbocity of the returned friends array
+	// 0 returns ONLY friend user IDs (default), 1 returns friend user IDs AND usernames
+	type ICreate struct {
+		Id          string `json:"id" binding:"required"`
```

Comment:
We might want to talk about the capitalization of Id, but so far we've (see auth_controller, etc) used `ID` and not `Id`. I think technically `ID` is the correct camel casing, but it should be OK to keep the json as lowercase `id`